script/slowest_pages

```diff
@@ -20,7 +20,7 @@
 ################################################################################
 
 FILE = ARGV[0] || File.expand_path("../log/production.log", __dir__)
-NUM  = ARGV[1].to_i || 10
+NUM  = ARGV[1].to_i
```

Comment:
Why this change?

Reply:
Similar to above. `to_i` should always be truthy, so `10` never happens.

Reply:
PS both of the above were flagged by the [Lint/UselessOr cop](https://docs.rubocop.org/rubocop/cops_lint.html#lintuselessor).